LOCAL_DEVELOPMENT_SETUP.md

```diff
@@ -60,13 +60,29 @@ make serve-front-end
 
 Navigate to `http://127.0.0.1:8080` with a web browser, and the site should be up and running locally with CSS and JavaScript working!
 
+## Logging database queries
+
+It can be helpful when working with code that makes database queries to see the SQL statements that get executed. You can switch on logging of _all_ executed SQL statements by uncommenting this line in `configure.swift`:
+
+```swift
+app.logger.logLevel = .debug
```

Comment:
This isn't quite what I'd intended with the original change. Let me recap that.

The default log level is `info` and the default sql logging is on `debug`. With the line that I added in `configure.swift`:

```swift
                                // Set this to .info to log SQL queries with the default log level.
                                sqlLogLevel: .debug),
```

I'm merely re-stating the default.

The instruction for users then is to change this to `.info`, in order to enable SQL logging on the default logging level of `info`.

The advantage here is that you don't need to lower the log level to `debug` and get all sort of other debug logging alongside the SQL that you're probably not interested in at that point.